modules/configuring-registry-storage-nutanix.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
+// * installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc
+// * installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc
+// * registry/configuring_registry_storage/configuring-registry-storage-nutanix.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-registry-storage-nutanix_{context}"]
+= Configuring registry storage for Nutanix
+
+As a cluster administrator, following installation you must configure your registry to use storage.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* A cluster on Nutanix. 
+* You have provisioned persistent storage for your cluster, such as {rh-storage-first}.
++
+[IMPORTANT]
+====
+{product-title} supports `ReadWriteOnce` access for image registry storage when you have only one replica. `ReadWriteOnce` access also requires that the registry uses the `Recreate` rollout strategy. To deploy an image registry that supports high availability with two or more replicas, `ReadWriteMany` access is required.
+====
++
+* You must have 100 Gi capacity.
+
+.Procedure
+
+. To configure your registry to use storage, change the `spec.storage.pvc` in the `configs.imageregistry/cluster` resource.
++
+[NOTE]
+====
+When using shared storage, review your security settings to prevent outside access.
+====
+
+. Verify that you do not have a registry pod:
++
+[source,terminal]
+----
+$ oc get pod -n openshift-image-registry -l docker-registry=default
+----
++
+.Example output
+[source,terminal]
+----
+No resourses found in openshift-image-registry namespace
+----
++
+[NOTE]
+=====
+If you do have a registry pod in your output, you do not need to continue with this procedure.
+=====
+
+. Check the registry configuration:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry.operator.openshift.io
+----
++
+.Example output
+[source,yaml]
+----
+storage:
+  pvc:
+    claim: <1>
+----
+<1> Leave the `claim` field blank to allow the automatic creation of an `image-registry-storage` persistent volume claim (PVC). The PVC is generated based on the default storage class. However, be aware that the default storage class might provide ReadWriteOnce (RWO) volumes, such as a RADOS Block Device (RBD), which can cause issues when replicating to more than one replica.
```

Comment:
```suggestion
<1> Leave the `claim` field blank to allow the automatic creation of an `image-registry-storage` persistent volume claim (PVC). The PVC is generated based on the default storage class. However, be aware that the default storage class might provide ReadWriteOnce (RWO) volumes, such as a RADOS Block Device (RBD), which can cause issues when you replicate to more than one replica.
```

The phrase "...which can cause issues when replicating to more than one replica" can be clearer. See [Verbs](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs) in IBM Style:
> When you use a verb phrase at the beginning of a sentence with a present participle, such as _creating_, or a past participle, such as _created_, make sure that the verb phrase modifies the correct word. Failure to do so results in a dangling modifier.